### ISSUE TITLE
Error when building

### ISSUE THREAD

**Issue by ricardoalcocer** [trueinteractions/tint2#177]
Hi,

This is the first time I try to package this app with the latest version.  Everything works in interactive mode, but I'm getting this error when building.  Any ideas?

```
[7051 of 13674] packaging node_modules/rimraf/README.md
Error
    at new q (/usr/local/bin/tntbuild:2890:3891)
    at N (/usr/local/bin/tntbuild:2890:3936)
    at p (/usr/local/bin/tntbuild:2890:5078)
    at Object.E [as input] (/usr/local/bin/tntbuild:2890:7368)
    at o (/usr/local/bin/tntbuild:2890:8727)
    at Object.H [as parse] (/usr/local/bin/tntbuild:2890:13448)
    at /usr/local/bin/tntbuild:2890:24690
    at Array.forEach (native)
    at Object.n.minify (/usr/local/bin/tntbuild:2890:24603)
    at Object.<anonymous> (/usr/local/bin/tntbuild:192:48)
Error: Unexpected character '#'
```

**Comment by ricardoalcocer** [comment]
Worked around it by setting "obsfucate":false on package.json...so it seems like a problem with uglifyjs.

**Comment by trevorlinton** [comment]
@ricardoalcocer It appears there's an odd bug in uglify where tabs against new lines are treated as a parser exception.  I'll leave this open until we figure out a solution.  The problem doesn't appear on the CLI execution, only when executing uglify as a module, which is also strange.  My hunch is the module version requires a config option to use a different AST parser than the command line uses. 

We'll keep you updated when there's a solution.

Thanks,
Trevor

**Comment by trevorlinton** [comment]
@ricardoalcocer This has been fixed on tint v2.2.4, due out tomorrow. Feel free to download the latest sources on master and rebuild if you'd like to try it sooner.
